features/step_definitions/user_steps.rb

```diff
@@ -18,6 +18,51 @@
     user = FactoryGirl.create(:user, hash)
     user.activate
     user.pseuds.first.add_to_autocomplete
+    step %{confirmation emails have been delivered}
+  end
+end
+
+Given /the following users exist with BCrypt encrypted passwords/ do |table|
+  table.hashes.each do |hash|
+    user = FactoryGirl.create(:user, hash)
+    user.activate
+    user.pseuds.first.add_to_autocomplete
```

Comment:
Hmmm... is this necessary? I'm not seeing autocomplete-related tests that use "the following users exist with BCrypt encrypted passwords." Ditto "the following users exist with SHA-512 encrypted passwords," actually.

Reply:
It's not but I'm kinda on the fence... is it worth being consistent? In the unlikely event that we (somehow) use this step to set up a test that involves autocomplete we're not left head-scratching why autocomplete doesn't work. That happened to me on https://github.com/otwcode/otwarchive/pull/2966/files#diff-904a4107d89f81a525345d37804626a9.

Reply:
Laziness wins: we'll keep it.